src/main/java/com/blogify/blogapi/repository/model/CommentReaction.java

```diff
@@ -0,0 +1,16 @@
+package com.blogify.blogapi.repository.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Entity
+@Table(name = "\"comment_reaction\"")
+@Data
+@Builder
+@AllArgsConstructor
+public class CommentReaction extends Reaction {
```

Comment:
Be careful with inheritance : you should find out what is `SuperBuilder` and `MappedSuperClass`